doc/userguide/rules/tag.rst

```diff
@@ -0,0 +1,129 @@
+Tag
+===
+
+The `tag` keyword allows tagging of the current and future packets.
+
+Tagged packets can be logged in `EVE` and conditional PCAP logging.
+
+Tagging is limited to a scope: `host` or `session` (flow). When using `host` a
+direction can be specified: `src` or `dst`. This selects tagging to happen by
+the `src` or `dst` IP address of the packet generating the alert.
+
```

Comment:
nit: I was thinking here instead of "This selects tagging.." maybe "Tagging will then occur based on.."?